stdlib/map.mli

```diff
@@ -86,9 +86,9 @@ module type S =
         @before 4.03 Physical equality was not ensured. *)
 
     val add_to_list: key -> 'a -> 'a list t -> 'a list t
-    (** [add_to_list key data m] is [m] with [key] mapped to [l] such
-        that [l] is [data :: Map.find key m] if [key] was bound in
-        [m] and [[v]] otherwise.
+    (** [add_to_list key data m] is [m] with [key] mapped to
+        [data :: Map.find key m] if [key] was bound in [m]
+        and [[data]] otherwise.
```

Comment:
For what it is worth, I find the formulation using `l` to make the scope of the `if` clearer. The change of `v` to `data` is an independent fix that should be made.

Reply:
I can agree with this